docs/main/updating/plugins/4-0.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Updating plugins to 4.0
+description: Guide for updating a Capacitor v3 plugin to v4
+slug: /updating/plugins/4-0
+---
+
+# Updating a Capacitor v3 plugin to 4.0
+
+There are only a few changes required to update a plugin from Capacitor 3 to Capacitor 4.
+
+:::info
+While you can follow the manual steps below, check out [this tool](https://github.com/rdlabo-team/capacitor-plugin-to-v4) from community member [@rdlabo](https://twitter.com/rdlabo) that automatically updates a Capacitor 3 plugin to v4.
+:::
+
+## Android
+
+### Update Android Project Variables
+
+In your `build.gradle` file, update your values to the following new minimums:
+
+```groovy
+minSdkVersion = 22
+compileSdkVersion = 32
+targetSdkVersion = 32
+androidxAppCompatVersion = '1.4.2'
+junitVersion = '4.13.2'
+androidxJunitVersion = '1.1.3'
+androidxEspressoCoreVersion = '3.4.0'
+```
+
+### Update to Gradle 7
+
+Adjust your Gradle project settings in `File > Project Structure > Project`. The Android Gradle Plugin Version should be 7.2.1 or later and the Gradle Version should be 7.4.2 or later. Apply these changes and run a gradle sync by clicking on the Elephant Icon in the top right of Android Studio.
```

Comment:
```suggestion
Adjust your Gradle project settings in `File > Project Structure > Project`. The Android Gradle Plugin Version should be 7.2.1 and the Gradle Version should be 7.4.2. Apply these changes and run a gradle sync by clicking on the Elephant Icon in the top right of Android Studio.
```
Better not use "or later" as newer versions could cause issues, stick to the ones we used in all plugins and Capacitor itself.